python/cuml/preprocessing/LabelEncoder.py

```diff
@@ -268,9 +270,25 @@ def inverse_transform(self, y: cudf.Series) -> cudf.Series:
 
         reverted = y._column.find_and_replace(ran_idx, self.classes_, False)
 
-        return cudf.Series(reverted)
+        res = cudf.Series(reverted)
+        return res
 
     def get_param_names(self):
         return super().get_param_names() + [
             "handle_unknown",
         ]
+
+    def _to_cudf_series(self, y):
```

Comment:
@tvdboom @lowener @dantegd Thank you all so much for the comments. I've made changes accordingly. Are there existing functions to replace this function? `def _to_cudf_series`

Especially when `y` is `string` datatype. It seems `input_to_cuml_array(y)` doesn't work:
```
/raid/data/ml/anaconda3/envs/cuml_dev/lib/python3.8/site-packages/cudf/core/column/column.py in __cuda_array_interface__(self)
   1025     @property
   1026     def __cuda_array_interface__(self):
-> 1027         raise NotImplementedError(
   1028             f"dtype {self.dtype} is not yet supported via "
   1029             "`__cuda_array_interface__`"

NotImplementedError: dtype object is not yet supported via `__cuda_array_interface__`
```